source/fundamentals/auth.txt

```diff
@@ -189,59 +189,104 @@ username and password to authenticate your user.
 
 .. important::
 
-   The MONGODB-AWS authentication mechanism is only available in MongoDB
+   The MONGODB-AWS authentication mechanism is available only in MongoDB
    versions 4.4 and later.
 
 The ``MONGODB-AWS`` authentication mechanism uses your Amazon Web Services
 Identity and Access Management (AWS IAM) credentials to authenticate your
 user.
 
-To specify the ``MONGODB-AWS`` authentication mechanism, perform the
-following:
+To connect to a MongoDB instance with ``MONGODB-AWS`` authentication enbaled,
+specify the ``MONGODB-AWS`` authentication mechanism.
 
-- Assign the ``AuthMechanism`` option the value ``MONGODB-AWS``
-- Assign the ``Username`` option the value of your ``accessKeyID``
-- Assign the ``Password`` option the value of your ``secretAccessKey``
+The driver checks for you credentials in the following sources in order:
 
-.. code-block:: go
-   :emphasize-lines: 3, 5-6
+1. Connection string
+#. Environment variables
+#. Web identity token file
+#. AWS ECS endpoint specified in ``AWS_CONTAINER_CREDENTIALS_RELATIVE_URI``
```

Comment:
Q: what is `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI`? An option, a property, an object,...?

Reply:
It's an environment variable - I added that to the bullet